docs/rfcs/component-universal-telemetry.md

```diff
@@ -27,7 +27,7 @@ be automatically captured for each kind of pipeline component.
 
 ## Attributes
 
-All signals should use the following attributes:
+Spans, data points, and log records should carry the following attributes:
```

Comment:
What is the equivalent for profiles?

Reply:
There's multiple things that can have attributes in the case of profiles: samples, mappings and locations. I guess we can put them all